### PR TITLE
changed react favicon

### PR DESCRIPTION
**Pull request by Utkarsh1504**
issue #13

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
@Utkarsh1504 please share screenshot.

**Review by iamdevvalecha — CHANGES_REQUESTED** [review]
just change logo.

**Comment by Utkarsh1504** [comment]
Shall I have to design a logo for the favicon?

**Comment by iamdevvalecha** [comment]
@Utkarsh1504 yes, we need t-picker favicon icon.

**Comment by Utkarsh1504** [comment]
favicon updated

**Favicon preview**
Screenshot (27)

**preview in browser**
Screenshot (28)

File: public/index.html [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="favicon.ico" />
+  <link rel="icon" href="%PUBLIC_URL%/t-favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Web site created using create-react-app" />
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <!--
```
Comment (iamdevvalecha) [inline public/index.html:12]: LGTM! but why extra line. and don't change indentation.

File: public/index.html [hunk 2]
```diff
@@ -24,12 +23,13 @@
...
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <title>T Picker</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+  <!--
```
Comment (iamdevvalecha) [inline public/index.html:32]: indentation

File: public/index.html [hunk 3]
```diff
@@ -39,5 +39,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```
Comment (iamdevvalecha) [inline public/index.html:44]: remove extra line and fix indent.

**Review by iamdevvalecha — CHANGES_REQUESTED** [review]

**Comment by iamdevvalecha** [comment]
just update favicon don't change anything. thank you.

**Comment by Utkarsh1504** [comment]
may be indentation changed due to my vscode setting. Okay i will fix it

**Comment by iamdevvalecha** [comment]
> may be indentation changed due to my vscode setting. Okay i will fix it

just change favicon. nothing else.

**Comment by Utkarsh1504** [comment]
ok fixed now

**Comment by iamdevvalecha** [comment]
@Utkarsh1504 thank you!

**Pull request merged**